pos_product_no_merge/static/src/js/db.js

```diff
@@ -0,0 +1,13 @@
+/*
+Copyright (C) 2023-Today: GRAP (<http://www.grap.coop/>)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+odoo.define('pos_tare.db', function (require) {
```

Comment:
🚫 blocking: without this fix, you will get unexpected errors if `pos_tare` is installed, as only one js file will be loaded -- can't have two with the same name 😓 

```suggestion
odoo.define('pos_product_no_merge.models', function (require) {
```

This file should probably be merged with `models.js`, as they're both modifying the core models.

Reply:
⚠️ this is still unresolved

Reply:
ow ! thanks !

Reply:
Fixed.